coreneuron/io/nrn_setup.cpp

```diff
@@ -635,21 +635,19 @@ void read_phasegap(NrnThread& nt, UserParams& userParams) {
 // only voltage, i_membrane_ or mechanism data index allowed. (mtype 0 means time)
 double* stdindex2ptr(int mtype, int index, NrnThread& nt) {
     if (mtype == voltage) {  // voltage
-        int v0 = nt._actual_v - nt._data;
-        int ix = index;  // relative to _actual_v
+        int ix{index};  // relative to _actual_v
         nrn_assert((ix >= 0) && (ix < nt.end));
         if (nt._permute) {
             node_permute(&ix, 1, nt._permute);
         }
-        return nt._data + (v0 + ix);    // relative to nt._data
+        return nt._actual_v + ix;
     } else if (mtype == i_membrane_) {  // membrane current from fast_imem calculation
-        int i_mem = nt.nrn_fast_imem->nrn_sav_rhs - nt._data;
```

Comment:
This was leading to segfaults when AddressSanitizer was enabled, as in those builds `nrn_sav_rhs` and `_data` (which are independent allocations) had addresses that were too different to fix in an `int`.

Reply:
👍 (didn't see any reason for previous logic anyway...)